container-registry/task-containerize.yaml

```diff
@@ -29,6 +29,9 @@ spec:
     - name: registry-namespace
       description: container registry namespace. required if no image-url or no image pipeline resources provided
       default: ""
+    - name: registry-create-namespace
+      description: create container registry namespace if it doesn't already exists
+      default: true
```

Comment:
We put default value as plain string in the YAML as we hit some issues with tekton definitions in the past for boolean value
So I would expect `default: "true"`